game-warhammer40kdarktide/index.js

```diff
@@ -1,33 +1,179 @@
-// Import some assets from Vortex we'll need.
-const path = require("path");
-const { fs, util } = require("vortex-api");
-
-// Nexus Mods domain for the game. e.g. nexusmods.com/warhammer40kdarktide
-const GAME_ID = "warhammer40kdarktide";
-// Steam app id
-const STEAM_APPID = "1361210";
-// Microsoft Store app id (gamepass)

```

Comment:
Can we keep these code comments?

Reply:
We can but for the STEAM_APPID  I don't use that name I use STEAMAPP_ID

Reply:
That's fine, just want the variables labelled so it's easier for other people to understand where the info comes from